db/migrate/20220811085951_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :title
+      t.text :text
+      t.integer :commentscounter
+      t.integer :likescounter
```

Comment:
- [ ] Good practice of creating vars is separating two words with an underscore, eg: `comments_counter`, `likes_counter`
This will help you in further development 😉

Reply:
Implemented 👍